best_practices/creating-the-project.rst

```diff
@@ -4,53 +4,45 @@ Creating the Project
 Installing Symfony
 ------------------
 
-In the past, Symfony projects were created with `Composer`_, the dependency manager
-for PHP applications. However, the current recommendation is to use the **Symfony
-Installer**, which has to be installed before creating your first project.
+.. best-practice::
+
+    Use Composer and Symfony Flex to create and manage Symfony applications.
+
+`Composer`_ is the package manager used by modern PHP application to manage their
+dependencies. `Symfony Flex`_ is a Composer plugin designed to automatize some
+of the most common tasks performed in Symfony applications. Using Flex is optional
+but recommended because it improves your productivity significantly.
 
 .. best-practice::
 
-    Use the Symfony Installer to create new Symfony-based projects.
+    Use the Symfony Skeleton to create new Symfony-based projects.
 
-Read the :doc:`/setup` article learn how to install and use the Symfony
-Installer.
+The `Symfony Skeleton`_ is a minimal and empty Symfony project which you can
+base your new projects on. Unlike past Symfony versions, this skeleton installs
+the absolute bare minimum amount of dependencies to make a fully working Symfony
+project. Read the :doc:`/setup` article to learn more about installing Symfony.
 
 .. _linux-and-mac-os-x-systems:
 .. _windows-systems:
 
 Creating the Blog Application
 -----------------------------
 
-Now that everything is correctly set up, you can create a new project based on
-Symfony. In your command console, browse to a directory where you have permission
-to create files and execute the following commands:
+In your command console, browse to a directory where you have permission to
+create files and execute the following commands:
 
 .. code-block:: terminal
 
     $ cd projects/
-    $ symfony new blog
-
-    # Windows
-    c:\> cd projects/
-    c:\projects\> php symfony new blog
-
-.. note::
-
-    If the installer doesn't work for you or doesn't output anything, make sure
-    that the `Phar extension`_ is installed and enabled on your computer.
+    $ composer create-project symfony/skeleton blog
 
 This command creates a new directory called ``blog`` that contains a fresh new
-project based on the most recent stable Symfony version available. In addition,
-the installer checks if your system meets the technical requirements to execute
-Symfony applications. If not, you'll see the list of changes needed to meet those
-requirements.
+project based on the most recent stable Symfony version available.
 
 .. tip::
 
-    Symfony releases are digitally signed for security reasons. If you want to
-    verify the integrity of your Symfony installation, take a look at the
-    `public checksums repository`_ and follow `these steps`_ to verify the
-    signatures.
+    The technical requirements to run Symfony are simple. If you want to check
+    if your system meets those requirements, read :doc:`/reference/requirements`.
```

Comment:
What about recommending using the `req-checker` for that?

Reply:
Agree! The linked article (https://symfony.com/doc/master/reference/requirements.html) was revamped recently to remove everything and just explain how to use req-checker.